### ISSUE TITLE
No video then note slide located at bottom of page

### ISSUE THREAD

**Issue by einhander** [Cimbali/pympress#309]
**Describe the bug**
Then `\setbeameroption{show notes on second screen=bottom}` used video can't be played.
`show notes on second screen=right` plays video ok.

**To Reproduce**
1. Use MWE:
```
\documentclass{beamer}
\usepackage{multimedia}
\setbeameroption{show notes on second screen=bottom} 
\begin{document}
\begin{frame}{Just a mp4 here}
    \centering
    \movie[width=0.3\textwidth]{\includegraphics[width=0.4\textwidth]{frame1}}{movie.mp4}
\end{frame}
\end{document}
```
2. Run pympress on result pdf file
4. Click on video frame
5. See no video playing, only sound

**Expected behavior**
Play video on presenter window

**Environment (please complete the following information):**
 - OS: Debian testing
 - Python version: 3.11.7
 - Pympress version: [e.g. 1.5.0]
 - Installation method: deb from debian repo

**Debug information (see below for file locations)**
 - What is reported in pympress.log?
```
INFO:pympress.app:Pympress: 1.8.4 ; Python: 3.11.7 ; OS: Linux 6.5.0-5-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.5.13-1 (2023-11-29) ; Gtk 3.24.40 ; GLib 2.78.3 ; Poppler 22.12.0 cairo ; Cairo 1.18.0 , pycairo 1.25.1
INFO:pympress.extras:Media support: GdkPixbuf gif player, GStreamer 1.22.8, VLC 3.0.20 Vetinari
```
 - Does the problem still happen if you remove your config file?
Yes

**Comment by Cimbali** [comment]
The clickable area to play the video appears in the notes slide. And the same bug happens with notes on the left, while notes on top works. There must be something wrong in the media position computation.

**Comment by einhander** [comment]
Yes, clickable area works on notes slide, I can hear sound from the video. But video didn't appear on any screen.

**Comment by Cimbali** [comment]
This seems to me to be a variant of #115 of some kind. Essentially beamer generates the wrong coordinates in the PDF for the media if you’re using the second* half (right or bottom) for content and the first half (left or right).

We can shift the media on the slide page when it is on a right or bottom notes page, under the rather safe assumption that nobody puts medias on notes pages. However the click area means shifting links which is less obvious: there is likely a valid use case for links on notes pages.

\* in the sense of latex/beamer coordinates

**Comment by Cimbali** [comment]
Frankly I think the most efficient workaround is to not use bottom- or left-screen note pages. My favourite option would be to display notes on separate pdf pages. Your example would become:

```latex
\documentclass{beamer}
\usepackage{multimedia}

% Show beamer notes but on separate pages
\setbeameroption{show notes}

% Tell pympress which pages are notes pages
\addtobeamertemplate{note page}{}{\thispdfpagelabel{notes:\insertframenumber}}

% Only add this bit if you want all note pages to show even when empty
\makeatletter
\def\beamer@framenotesbegin{% at beginning of slide
  \gdef\beamer@noteitems{}%
  \gdef\beamer@notes{{}}% used to be totally empty.
}
\makeatother

% Rest of document unchanged
\begin{document}
\begin{frame}{Just a mp4 here}
    \centering
    \movie[width=0.3\textwidth]{\includegraphics[width=0.4\textwidth]{frame1}}{movie.mp4}
\end{frame}
\end{document}
```

**Comment by einhander** [comment]
@Cimbali Thanks for the clarification of the source of the problem and the code snippet for the notes on separate pdf pages.
>if you’re using the second* half (right or top) for content and the first half (left or bottom) for notes.

Yes, the top position works fine. I'll stop there for now, unless there's a workaround. Unfortunately, the hack posted in #115 didn't work for me in case of bottom position.

**Comment by Cimbali** [comment]
The hack in 115 is for links. We’d need to expand the patch for `\movie`.
